website/docs/r/notification_destination.html.markdown

```diff
@@ -243,51 +243,15 @@ resource "newrelic_notification_destination" "foo" {
 
 In order to create a Slack destination, you have to grant our application access to your workspace. This process is [based on OAuth](https://api.slack.com/authentication/oauth-v2) and can only be done through a browser.
 As a result, you cannot set up a Slack destination purely with Terraform code.
-You can either create a Slack destination in the interface and [import](#import) it to Terraform, or use destination data source.
+You can either use [the destination data source](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/notification_destination),
+or reference destination id directly. Destination ID which can be found through [the destination data source](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/notification_destination),
+our GraphQL API, or using the user interface.
 
+~> **NOTE:** Please be aware that while you can import the Slack destination resource, it can only be used to delete it through Terraform.
```

Comment:
as discussed on Slack, this note can be shifted downwards, with a **warning** that states
 
```
Slack-based destinations configured/set up using the `newrelic_notification_destination` resource can only be imported and destroyed; this resource **does not** support creating/updating Slack-based destinations, owing to the reasons stated above, under the **Slack** section. Hence, **it is recommended to use the** `newrelic_notification_destination` data source instead, **to fetch the ID of the Slack destination configured via the UI** to use with other resources in the New Relic Terraform Provider, such as `newrelic_workflow`.
```